src/sentry/utils/samples.py

```diff
@@ -8,11 +8,20 @@
 from __future__ import absolute_import
 
 import os.path
+from datetime import datetime, timedelta
 
 from sentry.constants import DATA_ROOT
 from sentry.event_manager import EventManager
 from sentry.utils import json
 
+epoch = datetime.utcfromtimestamp(0)
+now = datetime.now()
```

Comment:
But this is always going to be relative from the time the process starts up until it reboots. If that's ok, ignore me. :)